### ISSUE TITLE
Create an "Everyone" sensor

### ISSUE THREAD

**Issue by danemacmillan** [PeteLawrence/homebridge-people#55]
There currently exist "Anyone" and "No one" sensors. An "Everyone" sensor would allow automations with HomeKit's `Security System` accessory that could arm the perimeter of an alarm system when everyone is home.

The typical use case would be when everyone is home and sleeping. Most alarm systems offer an "away" and "stay" arming configuration. For example, the former arms everything, including motion detectors, while the latter just arms door, window, and glass sensors, but excludes motion detectors, so people can still walk around the house.

**Comment by nitaybz** [comment]
@danemacmillan why not just setting configuration to work when everybody is "HOME"?

IF(trigger:) (XXX is "HOME") OR (YYY is "HOME") 
AND(condition:) ((XXX is "HOME") AND (YYY is "HOME")) 
THEN set "STAY" Mode in alarm system.

That way every time someone get in the house and detected as "HOME" it will trigger the automation that will check if every one is "HOME" before executing the command.

**Comment by danemacmillan** [comment]
@nitaybz Apple's Home app doesn't support complex evaluations. It's not possible to select multiple accessories--in this case being multiple occupancy sensors--otherwise I would have just selected all of the individual occupancy sensors and created the automation.

**Comment by nitaybz** [comment]
and that's why using Home app for creating automations are not the optimal way.
using "Eve" or "Hesperus"  apps (which are free) are much better to create complicated automations.

**Comment by danemacmillan** [comment]
@nitaybz I agree that the Home app is not the most powerful at the moment, but I'm using homebridge, like many others, to consolidate as much of the configuration into a single app. I could achieve what I want with all the individual apps, but I want a single point of control. I'm willing to hold out for future Home updates.

**Comment by nitaybz** [comment]
@danemacmillan you can definitely set all automation on each app but control all of the accessories only from one. unfortunately every app has it's downside and upside so using a combination of them would help you to achieve almost any kind of automation you wish for
